Cargo.toml

```diff
@@ -266,6 +266,11 @@ name = "ring"
 libc = "0.2.20"
 untrusted = "0.3.2"
 
+[build-dependencies]
+gcc = "0.3"
+target_build_utils = "0.2"
```

Comment:
Please change the target_build_dependencies dependency to `{ version = "0.2", default-features = false }` to reduce the dependencies.